docs/tx_rollups.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Transaction Optimistic Rollups
+id: tx_rollups
+author: Davis Sawali
+---
+
+A Transaction Optimistic Rollup (TORU) is an experimental temporary scaling solution to help Tezos developers and users get acquainted with the idea of working with rollups. 
+
+This is a trimmed, more concise documentation of how to do TORU operations in Taquito. If you aren't yet familiar with rollup node interactions, please refer to this [documentation](https://tezos.gitlab.io/alpha/transaction_rollups.html?highlight=transaction%20rollups#transaction-rollups) by Nomadic Labs.
+
+TORU currently supports the transferring of funds in the form of [tickets](https://tezostaquito.io/docs/tickets). Fund (or ticket) transfers can be done from:
+- Layer-1 to layer 2 (deposit)
+- Layer-2 to layer 2 (transfer)
+- Layer-2 to layer-1 (withdrawal)
+
+Taquito currently supports **layer-1** operations that facilitate deposits and withdrawals to and from the rollup node.
+
+## Deposits
+To be able to interact or transfer funds on layer-2, you will first need to deposit an amount to an existing layer-2 (tz4) address. 
+
+Depositing tickets from layer-1 to layer-2 can be done via a smart contract that facilitates the transfer to a tz4 address in a specified rollup node.
+
+Example of such contract can be found [here](https://tezos.gitlab.io/alpha/transaction_rollups.html?highlight=transaction%20rollups#depositing-assets-on-a-rollup)
+
+Assuming the contract have been originated, interacting with the contract in Taquito would look something like this:
+```
+const Tezos = new TezosToolkit('https://jakartanet.ecadinfra.com');
+
+const deposit = Tezos.contract.at(SMART_CONTRACT_ADDRESS);
+
+const op = await deposit.methods.default(
+    'foobar',
+    '15',
+    'tz4Jxn8MpRndqWUzkuZbQKmE3aNWJzYsSEso',
+    'txr1c9XcfmiLSWeqDPamPytns2ZXvwgYA7EpA'
+);
+
+const confirmation = await op.confirmation();
+
+console.log(op.hash);
+console.log(op.operationResults);
+```
+The `default` entrypoint takes in 4 arguments:
+- `foobar` is the ticket string
+- `15` is the quantity of a ticket string
+- `tz4Jxn8MpRndqWUzkuZbQKmE3aNWJzYsSEso` is the layer-2 address that will be the deposit recipient
+- `txr1c9XcfmiLSWeqDPamPytns2ZXvwgYA7EpA` is the rollup node id 
+
+If the deposit is successful, you will be returned an operation hash that can be accessed via `op.hash`
+
+You also might want to look at `op.operationResults` to retrieve the `ticket_hash`. A ticket hash should look something like this: `exprtz9FgfdzufUADVsvP8Gj8d8PZr9RsBEjZ5GQKM8Kp5cKWww7di`
+
+## Transfer
+Exchange of assets in the form of tickets can be done from a layer-2 address to another layer-2 address. Since transfer is a layer-2 operation, Taquito will not be supporting it. This may change in the future with SCORU (Smart Contract Optimistic Rollups).
```

Comment:
"Since transfer is a layer-2 operation" ... sure, but what about the transfer above? is it a different transfer?

Reply:
in this case a transfer from a tz4 to another tz4 address is a different operation from a `transfer_ticket` operation. I will add a disclaimer to not confuse the two. Thanks for pointing it out!